build-tools/automation/azure-pipelines.yaml

```diff
@@ -58,7 +58,7 @@ variables:
 - name: ExcludedNUnitCategories
   value: '& cat != DotNetIgnore & cat != HybridAOT & cat != MkBundle & cat != MonoSymbolicate & cat != PackagesConfig & cat != StaticProject & cat != SystemApplication'
 - name: DefaultTestSdkPlatforms  # Comma-separated SDK Platform(s) to install on test agents (no spaces)
-  value: 33
+  value: 33,UpsideDownCake
```

Comment:
UpsideDownCake is required for tests?

Reply:
Yes, there are some tests to ensure users can build with our unstable packages.